tol_lab_share/config/rabbit.py

```diff
@@ -0,0 +1,44 @@
+import os
+from lab_share_lib.config.rabbit_config import RabbitConfig
+from lab_share_lib.config.rabbit_server_details import RabbitServerDetails
+
+from tol_lab_share.constants import (
+    RABBITMQ_SUBJECT_BIOSCAN_POOL_XP_TO_TRACTION,
+    RABBITMQ_SUBJECT_CREATE_LABWARE,
+    RABBITMQ_SUBJECT_UPDATE_LABWARE,
+)
+from tol_lab_share.processors.bioscan_pool_xp_to_traction_processor import BioscanPoolXpToTractionProcessor
+from tol_lab_share.processors.create_labware_processor import CreateLabwareProcessor
+from tol_lab_share.processors.update_labware_processor import UpdateLabwareProcessor
+
+RABBIT_SERVER_DETAILS = RabbitServerDetails(
+    uses_ssl=False,
+    host=os.environ.get("LOCALHOST", "127.0.0.1"),
```

Comment:
Minor comment: is `LOCALHOST` the right term for this? Maybe `HOST`?

Reply:
This is how it was set up years ago.  The reason it's defined this way is so that when you're running the app on your local dev environment, it does actually point at `localhost` but when it's in a Docker environment, you use the Docker reference back to your host machine.

See https://github.com/sanger/tol-lab-share/blob/ab56e3801d0aa972ca30d48228fb67d8324e1b63/tol_lab_share/config/defaults.py#L11